content/SCALE/SCALETutorials/Apps/CommunityApps/WebDAV.md

```diff
@@ -1,109 +1,169 @@
+| WEBDRIVE_PASSWORD_FILE | Use to specify a file that contains the password instead of using the **Password** field. Use when **Authentication Type** is set to **Basic Authorization**. |
+| WEBDRIVE_MOUNT | Use to specify the location within the container where to mount the WebDAV resource (drive) into the container. This defaults to /mnt/webdrive and is not meant to be changed. |
+{{< /truetable >}}
+{{< /expand >}}
 
-The **truenas_default Certificate** can be used as the **Certificate** when no other specific certificates are available.
+### User and Group Configuration Settings
+The default user and group for WebDAV is 666. To specify a different user, create the user and group before installing the application, then enter the user ID (UID) and group ID (GID) in the fields for these settings. 
 
-### Share Settings
+{{< trueimage src="/images/SCALE/23.10/InstallWebDAVUserAndGroupConfig.png" alt="User And Group Configuration" id="6: User And Group Configuration" >}}
 
-Add your existing WebDAV share to the application.
-Click **Add** to the right of **Shares** in the **Storage Configuration** section.
+### Network Configuration Settings
+The container for the WebDAV app has **Enable HTTP** selected by default. The port for HTTP is 30034.
 
-{{< trueimage src="/images/SCALE/22.12/WebDAVStorageConfigAddShare.png" alt="WebDAV Storage Configuration Add Share" id="4: WebDAV Storage Configuration Add Share" >}}
+{{< trueimage src="/images/SCALE/23.10/InstallWebDAVAppNetworkConfig.png" alt="WebDAV Network Configuration for HTTP" id="7: WebDAV Network Configuration for HTTP" >}}
 
-Enter the **Share Name**.
-This can be the share name recorded from a configuration in **Shares > WebDAV**.
+To add encryption to the web traffic between clients and the server, clear the checkmark in **Enable HTTP** and select **Enable HTTPS**.
+This changes the default port in **HTTPS Port** to 30035, and adds a system **Certificate**. 
 
-A share **description** is not required by can be a useful place to record notes about the share.
+{{< trueimage src="/images/SCALE/23.10/InstallWebDAVNetworkConfigEnableHTTPS.png" alt="WebDAV Network Configuration for HTTPS" id="8: WebDAV Network Configuration for HTTPS" >}}
 
-Enter or browse to the **Host Path** location for the WebDAV share dataset.
-Use a path previously used in a saved configuration from **Shares > WebDAV**.
+The default certificate is 0. You can use the default as the **Certificate** if no other specific certificate is available. 
 
-Select **Read Only** to make the share read only and disable write access to the share.
+### Storage Configuration Settings
+You can add one or more shares in the **Storage Configuration** section. 
```

Comment:
Again wondering if we should specify that at least one share is required.

If you don't add at least one before clicking Install it displays the error "Empty value not allowed"